Modules/Quotas/test/Quotas.Application.Tests/Tests/Quotas/DeleteIndividualQuota/HandlerTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Backbone.Modules.Quotas.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Quotas.Application.Tiers.Commands.DeleteQuotaForIdentity;
+using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using Backbone.Modules.Quotas.Domain.Aggregates.Tiers;
+using Enmeshed.BuildingBlocks.Application.Abstractions.Exceptions;
+using Enmeshed.BuildingBlocks.Domain;
+using Enmeshed.UnitTestTools.Extensions;
+using FakeItEasy;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using Xunit;
+using Handler = Backbone.Modules.Quotas.Application.Tiers.Commands.DeleteQuotaForIdentity.Handler;
+using MetricKey = Backbone.Modules.Quotas.Domain.Aggregates.Metrics.MetricKey;
+
+namespace Backbone.Modules.Quotas.Application.Tests.Tests.Quotas.DeleteIndividualQuota;
+public class HandlerTests
+{
+    [Fact]
+    public async Task Delete_individual_quota()
+    {
+        // Arrange
+        var tierId = new TierId("SomeTierId");
+        var identityAddress = "some-identity-address";
+        var identity = new Identity("some-identity-address", tierId);
+        var createdQuota = identity.CreateIndividualQuota(MetricKey.NumberOfSentMessages, 5, QuotaPeriod.Month);
+        var command = new DeleteQuotaForIdentityCommand(identityAddress, createdQuota.Id);
+        var identitiesRepository = A.Fake<IIdentitiesRepository>();
```

Comment:
Can you please at least inline all variables that are only used once throughout the test? I hate this super long arrange block. It's hard to find the interesting part

Reply:
Should I also inline things like the creation of the quota?

Reply:
That's probably too much. I would just inline atomic values or things that are created by a short constructor, like value objects.